src/ui/RealmBrowser/Content/SingleObjectCollection.ts

```diff
@@ -0,0 +1,88 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+export class SingleObjectCollection<T extends Realm.Object> extends Array<T>
```

Comment:
I think this class needs a doc comment, since it's kind of funky that it's implementing `Realm.Results`.

Reply:
Added.